dist/yaml/ovn-setup.yaml

```diff
@@ -0,0 +1,189 @@
+# ovn-setup.yaml
+#
+# Setup for Openshift to support the ovn-cni plugin
+#
+# Create the project, service account and policies.
+# ovnkube interacts with kubernetes and the environment
+# must be properly set up.
+# 
+# This provisioning is done as part of installation after the cluster is
+# up and before the ovn daemonsets are created.
+
+---
+apiVersion: project.openshift.io/v1
+kind: Project
```

Comment:
For kubernetes, a namespace object is sufficient. We should split this file into two - one for plain kubernetes, and another for openshift-overrides.

Reply:
@rajatchopra All but the last yaml are openshift.io/v1. Are you thinking of putting the last item into another file?

Reply:
@pecameron instead of Project, just do a normal Kubernetes "Namespace"